packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -905,4 +905,33 @@ whisperkit-cli transcribe --audio-path /path/to/audio.mp3
 # Or use your preferred model variant
 whisperkit-cli transcribe --model "large-v3" --model-prefix "distil" --audio-path /path/to/audio.mp3 --verbose`,
 ];
+
+export const diffusionkit = (): string[] => [
+	`# Install CLI with pip
+pip install diffusionkit-cli
+
+# View all available options
+diffusionkit-cli --help
+
+# Generate image using default FLUX.1-schnell and save it to out.png
+diffusionkit-cli --prompt "a beautiful sunset over the ocean"
+
+# To use Stable Diffusion 3 accept the terms before downloading the checkpoint: https://huggingface.co/stabilityai/stable-diffusion-3-medium
+# Once you accept the terms, sign in with your Hugging Face hub token with read access to contents of all public gated repos you can access:
+huggingface-cli login --token YOUR_HF_HUB_TOKEN
+
```

Comment:
The "code snippet" part is generally a very short snippet to help users getting started. It should not contain installation or configuration steps and should be focused on "how to use **this model** with diffusionkit". Code snippets are shown on each model page and are customized to be ready to use.

```suggestion
    `
```

Reply:
Agree with @Wauplin. In addition, I'd recommend that different examples or use cases are returned as multiple strings, like in the [audioseal example](https://github.com/huggingface/huggingface.js/blob/4f9c8d32a0dfb46869afc40ed10866a947198c09/packages/tasks/src/model-libraries-snippets.ts#L49). This way snippets are separated visually and can be copied independently:

<img width="956" alt="Screenshot 2024-08-27 at 11 21 53" src="https://github.com/user-attachments/assets/bbd839cd-d12f-4cb4-96db-47def4062a43">

Reply:
Even better yes!

Reply:
Great example, was curious how to do this!